pkgs/development/tools/misc/libtool/libtool2.nix

```diff
@@ -15,6 +15,9 @@ stdenv.mkDerivation rec {
   nativeBuildInputs = [ perl help2man m4 ];
   propagatedBuildInputs = [ m4 ];
 
+  # FIXME: Configure script is too old (2015) to be able to guess RiscV platform.
+  configureFlags = lib.optional stdenv.buildPlatform.isRiscV "--build=${stdenv.buildPlatform.config}";
```

Comment:
Huh, are you sure this is necessary? Doesn't https://github.com/NixOS/nixpkgs/blob/32c2087987ac9d4f13d80780d97ee6038ffacdc9/pkgs/stdenv/linux/default.nix#L391-L392 and similar logic make this unnecessary?